specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2024-07-01/availabilitySet.json

```diff
@@ -431,6 +431,10 @@
           "$ref": "../../../common-types/v1/common.json#/definitions/SubResource",
           "description": "Specifies information about the proximity placement group that the availability set should be assigned to. Minimum api-version: 2018-04-01."
         },
+        "scheduledEventsPolicy": {
```

Comment:
could we put this at the end of the properties list for AvailabilitySetProperties.
It helps for SDK generation, putting the new param at the end of param change, avoiding change in param signature